packages/components/src/components/input-field/input-field.jsx

```diff
@@ -163,18 +164,24 @@ const InputField = ({
 
             decrement_value = parseFloat(getClampedValue(new_value)).toFixed(decimal_places);
         } else if (is_crypto || (!currency && is_float)) {
-            const new_value = parseFloat(+(current_value || 0)) - parseFloat(1 * 10 ** (0 - decimal_places));
-            decrement_value = parseFloat(new_value).toFixed(decimal_places);
+            const new_value =
+                parseFloat(+(current_value || 0)) -
```

Comment:
```suggestion
                (parseFloat(current_value) || 0) -
```
Why are we coercing `current_value` in a `parseFloat`? Wouldn't the above be cleaner?

Reply:
Will make the changes